IGI.Lab3/CarPark.WebAPI/Models/JwtToken.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.Authentication.JwtBearer;
+
+namespace CarPark.WebAPI.Models
+{
+    public class JwtToken
+    {
+        public const string Issuer = "CarPark";
```

Comment:
Move constants to a separate file, or to app settings.

Reply:
fix